packages/ag-charts-website/astro.config.mjs

```diff
@@ -118,6 +131,5 @@ export default defineConfig({
             skip: CHECK_REDIRECTS !== 'true',
         }),
     ],
-    experimental: { contentCollectionCache: EXPERIMENTAL_ASTRO_BUILD_CACHING === 'true' },
```

Comment:
Is this now enabled by default, and/or are there other caching implications here that we need to be aware of? (E.g. how does this play with the Nx cache?)

Reply:
This has been removed in v5: https://docs.astro.build/en/guides/upgrade-to/v5/#experimental-flags
I presumed it was being handled by the vite 6 upgrade, but I found your commit (https://github.com/ag-grid/ag-charts/commit/24195dbdf205196e31be03fa6510ac10d1b78927) related to it, and need to understand what it was about.